target/linux/mediatek/dts/mt7622-ubnt-unifi-6-lr-v2.dtsi

```diff
@@ -0,0 +1,19 @@
+#include "mt7622-ubnt-unifi-6-lr.dtsi"
+
+/ {
+	gpio-leds {
+		compatible = "gpio-leds";
+
```

Comment:
@dangowrt can you help me our with this section? this pretty much the board vendor dts, but not sure how it should be in OpenWRT.

Reply:
anyone more versed in OpenWRT that can help out with the labels here? do we need `linux,default-trigger`? used in the vendor dts.

Any changes needed in `target/linux/mediatek/mt7622/base-files/etc/board.d/01_leds`?

Reply:
got it to work, and the right LED is updated when fiddling with `/sys/class/leds` for `white:dome` and `blue:dome`

Reply:
I'd set `linux,default-trigger = "default-on";` for one of them.
On those ubnt devices we usually don't use the dome leds to indicate traffic or wireless activity because they are usually in very visible positions and having them flash all the time makes people nervous... (users can of course still manually set led triggers if that's what they like)

Reply:
is this not covered by `led-running = &led_blue;` i.e. it should be blue when running? but possibly it would be best to then have `default-state = "off";` for the blue led to make sure that its really the running state change that changed the led? What do you think?

Reply:
but maybe this is more "safe"
```diff
                led_white: dome_white {
                        label = "white:dome";
                        linux,default-trigger = "default-on";
                        gpios = <&pio 0x43 GPIO_ACTIVE_HIGH>;
                };

                led_blue: dome_blue {
                        label = "blue:dome";
                        default-state = "off";
                        gpios = <&pio 0x44 GPIO_ACTIVE_HIGH>;
                };
```

then the white will always be on, and if the system get up to running we would switch to blue + white. The other way around is not that good, as it difficult to detect the white if the blue is already on.

Reply:
now included in fd87afb96e50c8eae7c2b408f4562d5e575556fa